src/main/java/org/alliancegenome/mati/controller/SubdomainController.java

```diff
@@ -0,0 +1,41 @@
+package org.alliancegenome.mati.controller;
+
+import io.quarkus.security.Authenticated;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.alliancegenome.mati.service.SubdomainService;
+import org.eclipse.microprofile.openapi.annotations.enums.SecuritySchemeType;
+import org.eclipse.microprofile.openapi.annotations.security.SecurityRequirement;
+import org.eclipse.microprofile.openapi.annotations.security.SecurityScheme;
+import org.eclipse.microprofile.openapi.annotations.security.SecuritySchemes;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@AllArgsConstructor
+@Slf4j
+@Path("/subdomain")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Tag(name = "Subdomain", description = "subdomain Operations")
+@SecuritySchemes(value = {
```

Comment:
If you put the @SecuritySchemes in a Application Class then you won't have to define it on all the endpoint classes.

Reply:
See how we do that in the RestApplication.java file in the curation system.

Reply:
Done!